server.js

```diff
@@ -31,8 +31,22 @@ require('./routes/api/donate-routes.js')(app);
 require('./routes/api/inventory-routes.js')(app);
 
 // Syncing our database and logging a message to the user upon success
-db.sequelize.sync().then(() => {
-  app.listen(PORT, () => {
-    console.log('==> 🌎  Listening on port %s. Visit http://localhost:%s/ in your browser.', PORT, PORT);
+// db.sequelize.sync().then(() => {
+//   app.listen(PORT, () => {
+//     console.log('==> 🌎  Listening on port %s. Visit http://localhost:%s/ in your browser.', PORT, PORT);
+//   });
+// });
+
+db.User.sync().then(() => {
```

Comment:
curious why we needed to nest all these?

Reply:
we do not - it was an attempt to get foreign keys working - thanks